cmd/apmsoak/run.go

```diff
@@ -67,7 +68,12 @@ func NewCmdRun() *cobra.Command {
 				logger.Fatal("Fail to initialize runner", zap.Error(err))
 			}
 
-			if err := runner.Run(cmd.Context()); err != nil {
+			// Graceful shutdown driven by SIGINT.
+			// Nothing else is expected to stop the runner.
+			ctx, cancel := signal.NotifyContext(cmd.Context(), os.Interrupt)
```

Comment:
what about SIGTERM?

Reply:
Good point! You are also hinting at the right signal sent by k8s shutdown, will add